### ISSUE TITLE
Feature request for image editor - Vertical text

### ISSUE THREAD

**Issue by stephen147** [ShareX/ShareX#4616]
It would be nice to have the option to either rotate existing text or create vertical text out of the box as opposed to a carriage return to do the same.

**Comment by stephen147** [comment]
Keen on this feature so bumping this thread.

**Comment by stephen147** [comment]
Bump

**Comment by berkdemir** [comment]
Agreed.

**Comment by alekseyshaklunov** [comment]
+

**Comment by Nap4lm** [comment]
Highly desired. Add a little rotation-symboled-anchor somewhere on the boxes of images/text-blocks that lets us rotate our objects :)

**Comment by maleko48** [comment]
agreed. text rotation would be great so I can position text to follow arrows along their length

**Comment by Athrish** [comment]
This would add to productivity of the users. Please add this rotate text feature.

**Comment by Lewis8379** [comment]
I was shocked today to find there's no ability to rotate text.

**Comment by helmanfrow** [comment]
> I was shocked today to find there's no ability to rotate text.

Same. Please add this feature.

**Comment by marchershey** [comment]
Bump bump bump. 

Need this feature!

**Comment by ToNyRANDRIAMANANTSOA** [comment]
This is indeed very useful. 
For now as a workaround I rotate and save the image, then edit the saved image and insert text and save, and then restore rotation.
Have a rotating feature for any angle would be essential.

**Comment by Devar** [comment]
Agreed, having a rotate handle or degree input for text (outline in particular) would be fantastic. Please!

**Comment by Alekseev2014s** [comment]
Please add this feature!

**Comment by NotolabrusFisicola** [comment]
![Image](https://github.com/user-attachments/assets/60b48791-8c43-4a3d-9b1e-0f05adbadca2)

I second that. I never had the need to rotate a text while taking a screenshot until earlier today, and was quite surprised to find out that we cannot!

**Comment by Zizouillo** [comment]
![Image](https://github.com/user-attachments/assets/a7c8dcf1-0fec-48ee-9547-b18d1023aea5)

Pleaase, think to add it

**Comment by TomVer99** [comment]
I would also like to express my desire for this feature.

**Comment by shodanx2** [comment]
Gang I think we will have to do it ourselves, maybe fix the win10 HDR bug too

Is anyone here good at vibecoding ?

**Comment by Pavel-Sushko** [comment]
+1

**Comment by eduardosouza84** [comment]
+1
